sklearn/metrics/_classification.py

```diff
@@ -632,12 +632,12 @@ class labels [2]_.
 
     References
     ----------
-    .. [1] J. Cohen (1960). "A coefficient of agreement for nominal scales".
+    .. [1] :doi:`J. Cohen (1960). "A coefficient of agreement for nominal scales".
            Educational and Psychological Measurement 20(1):37-46.
-           doi:10.1177/001316446002000104.
+           <10.1177/001316446002000104>`
     .. [2] `R. Artstein and M. Poesio (2008). "Inter-coder agreement for
-           computational linguistics". Computational Linguistics 34(4):555-596
-           <https://www.mitpressjournals.org/doi/pdf/10.1162/coli.07-034-R2>`_.
+           computational linguistics". Computational Linguistics 34(4):555-596.
+           <https://www.mitpressjournals.org/doi/pdf/10.1162/coli.07-034-R2>`
```

Comment:
Please revert this one: there is no `:doi:` directive so you need an underscore because this is a [sphinx external link](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links)